api/src/test/java/org/openmrs/OpenmrsTestsTest.java

```diff
@@ -111,18 +109,16 @@ public void shouldHaveClassNameEndWithTestIfContainsMethodTestAnnotations() {
 		for (Class<?> currentClass : getClasses("^.*(?<!Test|IT|PT)\\.class$")) {
 			
 			// skip over classes that are @Ignore'd
-			if (currentClass.getAnnotation(Ignore.class) == null) {
+			if (currentClass.getAnnotation(Ignore.class) == null || currentClass.getAnnotation(Disabled.class) == null) {
```

Comment:
oh, now I get it. So here it says `// skip over classes that are @Ignore'd`. So what we have to do is skip over classes that do not have the annotation `Ignore` **and** also do not have the annotation `Disabled`

```suggestion
			if (currentClass.getAnnotation(Ignore.class) == null && currentClass.getAnnotation(Disabled.class) == null) {
```

Otherwise you are going to check `CreateInitialDataSet`.

Reply:
So you get the error after my suggested change only because you are in fact checking a class that should not be checked since it is annotated with `@Ignore`. So you need to take both my suggestions and it should work if I am not mistaken.